example/android/app/src/main/AndroidManifest.xml

```diff
@@ -21,6 +21,12 @@
                 <category android:name="android.intent.category.LAUNCHER"/>
             </intent-filter>
         </activity>
+<!--        TODO Find a way to declare this service in a different manifest?/ provided by plugin -->
```

Comment:
If you haven't already then could you please create an issue to track this. Otherwise, `TODO` statements in the code like this become a parallel backlog.

Reply:
Comment removed in 41641c582c0fa4d23929718e9feda1772ea7aa38

There is no issue needed because it is not possible. Other libraries are making flutter developers modify the Android Manifest to add the service. For example, [`flutter_local_notifications` example `AndroidManifest.xml`](https://github.com/MaikuB/flutter_local_notifications/blob/master/flutter_local_notifications/example/android/app/src/main/AndroidManifest.xml).

Reply:
Here's something on resolution of `intent-filter` between manifests https://developer.android.com/studio/build/manifest-merge

>  _The `intent-filter` element is never matched between manifests. Each is treated as unique and is added to the common parent element in the merged manifest_

Reply:
Thanks, I did not know about [that page](https://developer.android.com/studio/build/manifest-merge), I was reading [another page](https://android-doc.github.io/tools/building/manifest-merge.html#merge-rules). I learnt about the Merged Manifest tab in Android Studio. I've successfully moved the declaration to the plugin AndroidManifest, out of the example. 945d57beb5eb4a2afdf909b03264d78a198fdae8